### DISCUSSION TITLE
How to use it on site subfolders and pages?

### DISCUSSION THREAD

**Discussion by MahmodMenawy** [Dhruv-Techapps/acf-docs#358] [Q&A]
Ex: like.com  
Here you press the button "Enter" 
leads to 
Like.com/run
Here you press "Next" 
Leads to
Like.com/run/fast
Here you press "Next" 
Leads to 
Like.com/walk
Here you press "submit"

I only have access to the main URL so I can't make different configurations for each page
I want to make one configurations from the main URL and keeps working on the other site pages. 

So for the example, only the command to press enter on the "Like.com" works, then the configuration stops.
Any help?

**Comment by Dhruv-Tech-Apps** [comment]
Hi @MahmodMenawy ,

You can make use of Skip https://getautoclicker.com/docs/3.x/action-settings/error-handling/ under action setting this way. if particular button is not found it will move to next action,

One configural with multiple action 
- to  click on enter
- to click on next
